index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+       <div class="logo"> <a><img src="images/image_geometry_1.png" alt="logo image"></a></div>
+        <nav>
+            <div class="Button">≡</div>
+        </nav>
+    </header>
+    <main>
+        <h1 class="intro">
+            Javed <br> Shahbaz
+        </h1>
+        <section class="paragraph">
+            <p>
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
```

Comment:
- [OPTIONAL] I think it's better to avoid lines of code longer than 120 characters. Your code will be more readable and follow the best practices. As not required by the previous reviewer, I consider it optional.
- [OPTIONAL] Kindly use the styling provided by the template for your fonts. Will create a harmonious UI.

<img width="871" alt="Screen Shot 2022-12-02 at 12 10 13 AM" src="https://user-images.githubusercontent.com/47673311/205206841-3e9cccf4-8781-454b-a794-a0aab0982e60.png">